db/migrate/20220331164557_create_transfer_packages.rb

```diff
@@ -0,0 +1,20 @@
+class CreateTransferPackages < ActiveRecord::Migration
+  def up
+    create_table :transfer_packages, id: false do |t|
+      t.string :id,                      limit: 36, null: false
+      t.string :receiver_institution_id, limit: 36, null: false
+      t.string :recipient
+
+      t.index :id, unique: true
```

Comment:
We can't seem to generate a `:string` primary key with ActiveRecord 4. It seems we be pass `:id => :string` with AR 5.0, but for now, I couldn't find a proper solution with AR4 :cry: 

Maybe I should follow the rest of the models: using integer ids for internal relationships, and have a distinct UUID column.

Reply:
I did just that: we now have an `id` (int) and an `uuid` (string) columns.